coralproject/talk
# [next] DOM Purification

## What does this PR do?

- Turns HTTP links on the page into clickable anchor tags
- Purifies comment bodies to strip undesired HTML entities and attributes
- Improved comment link detection

## How do I test this PR?

- Submit a comment directly via the API with `<script>`, or other tags, and see that they are stripped out when submitted
- See also that any anchor tags have a `rel="noopener noreferrer"` and `target="_blank"` set
